packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -170,11 +170,21 @@ export async function meterResponse(
       logger.info({ res }, 'Response sent')
     })
     .catch((err) => {
-      logger.error(ErrorMessage.UNKNOWN_ERROR)
-      logger.error(err)
+      logger.error({ err }, 'Caught error in outer endpoint handler')
```

Comment:
note: after investigating that weird req/res type discrepancy, this handling is the same in the signer/combiner; happy to extract this into a function in the common package though also not really sure it's super necessary. No strong opinion here!

Reply:
I agree it's probably not super necessary. This seems like a reasonable place to have duplicate code